rwengine/src/script/modules/GTA3ModuleImpl.inl

```diff
@@ -4978,35 +4979,43 @@ void opcode_01b4(const ScriptArguments& args, const ScriptPlayer player, const S
 }
 
 /**
-    @brief force_weather %1d%
+    @brief FORCE_WEATHER
 
-    opcode 01b5
-    @arg arg1 
+    opcode 01B5 / 437
+
+    Description
+
+    @arg weatherID Weather ID
 */
-void opcode_01b5(const ScriptArguments& args, const ScriptWeather arg1) {
-    RW_UNIMPLEMENTED_OPCODE(0x01b5);
-    RW_UNUSED(arg1);
-    RW_UNUSED(args);
+void opcode_01b5(const ScriptArguments& args, const ScriptWeather weatherID) {
+    args.getState()->basic.forcedWeather = weatherID;
 }
 
 /**
-    @brief set_weather %1d%
+    @brief FORCE_WEATHER_NOW weatherID
+
+    opcode 01B6 / 438
+
+    Description
 
-    opcode 01b6
     @arg weatherID Weather ID
 */
 void opcode_01b6(const ScriptArguments& args, const ScriptWeather weatherID) {
+    args.getState()->basic.lastWeather = weatherID;
     args.getState()->basic.nextWeather = weatherID;
+    args.getState()->basic.forcedWeather = weatherID;
 }
 
 /**
-    @brief release_weather
+    @brief RELEASE_WEATHER
+
+    opcode 01B7 / 439
```

Comment:
Again, unnecessary decimal + renamed opcode to something which is not giving any benefit. Please review other instances too.

Reply:
Documentation seemed messy so I basically tried to mess around with it.